routes/users.js

```diff
@@ -10,5 +10,19 @@ router.get("/users", async (req, res) => {
   res.send(users);
 });
 
+router.put("/users/:id", async (req, res) => {
+  const x = parseInt(req.params.id);
+  if(isNaN(x)) {
+    res.status(400).send({"error": "Invalid User ID"});
+  }
+  // get user by id
+  var elementOfArray = users.find(item => item.id === x);
+
+  // update email address
+  const newemail = req.body.email;
```

Comment:
should we change the const to var since the 'newemail' variable is being changed?